en/components/drop_down_virtual.md

```diff
@@ -0,0 +1,160 @@
+ - `containerSize` - the size of the virtualized container (in `px`). This needs to be enforced on the wrapping `<div>` as well
+ - `itemSize` - the size of the items that will be displayed (in `px`)
+
+In order to assure uniqueness of the items, pass `item` inside of the [`value`]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html#value) input and `index` inside of the [`index`]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html#index) input of the `igx-drop-down-item`.
+To preserve selection while scrolling, the drop down item needs to have a reference to the data items it is bound to.
+> [!NOTE]
+> For the drop down to work with a virtualized list of items, [`value`]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html#value) and [`index`]({environment:angularApiUrl}/classes/igxdropdownitemcomponent.html#index) inputs **must** be passed to all items.
+> [!NOTE]
+> It is strongly advised for each item to have an unique value passed to the `[value]` input. Otherwise, it might lead to unexpected results (incorrect selection). 
+> [!NOTE]
+> When the drop down uses virtualized items, the type of [`dropdown.selectedItem`]({environment:angularApiUrl}/classes/igxdropdowncomponent.html#selecteditem) becomes `{ value: any, index: number }`, where `value` is a reference to the data item passed inside of the `[value]` input and `index` is the item's index in the data set
+
+
+### Component Definition
+
+Inside of our component's `constructor`, we'll declare a moderately large list of items (container both headers and disabled items) which we'll be displaying in our drop-down. We also need to declare `itemHeight` and `itemsMaxHeight` (used in the template) inside of our `drop-down-virtual.component.ts` file:
+
+```typescript
+// drop-drop-virtual.component.ts
+export class DropDownVirtualComponent {
+  public items: DataItem[];
+  public itemHeight = 48;
+  public itemsMaxHeight = 480;
+
+  constructor(protected cdr: ChangeDetectorRef) {
+    const itemsCollection: DataItem[] = [];
+    for (let i = 0; i < 50; i++) {
+        const series = (i * 10).toString();
+        itemsCollection.push({
+            id: series,
+            name: `${series} Series`,
+            header: true,
+            disabled: false
+        });
+        for (let j = 0; j < 10; j++) {
+            itemsCollection.push({
+                id: `${series}_${j}`,
+                name: `Series ${series}, ${i * 10 + j} Model`,
+                header: false,
+                disabled: j % 9 === 0
+            });
+        }
+    }
+    this.items = itemsCollection;
+  }
+}
+```
+
+### Styles
+
+The last (but very important) part of the configuration happens inside of our component's style sheet, `drop-down-virtual.component.scss`. We need to declare that our wrapping div (`drop-down-virtual-wrapper`) has `overflow: hidden`, so that we don't end up with two scroll bars (one from the `igxFor` and one from the container itself):
```

Comment:
just a suggestion: "we need do declare" -> It's required to set the wrapping div ..... with overflow: hidden. This extra configuration prevents two scrollbars appearance (one from the .....)

Reply:
I've reworded it to be a bit clearer, changes are in the latest commit